src/database/parse-server.ts

```diff
@@ -0,0 +1,48 @@
+import { DatabaseConfig } from "./database-config";
+
+export default class ParseServer implements Database {
+  private static instance: Database;
+
+  private Parse: any;
+  private Event: any;
+
+  public static getInstance(Parse: any, AsyncStorage: any): Database {
+    if (this.instance == null) {
+      this.instance = new ParseServer(Parse, AsyncStorage);
+    }
+
+    return this.instance;
+  }
+
+  public async getEvents(
+    numResults: number,
+    pageNumber: number,
+    sortBy: string
+  ): Promise<any> {
```

Comment:
Example use:
```
// Get first 10 events sorted by ascending date
function getEventsFromDB() {
  const events = Database.getEvents(10, 1, "dateTime");
  events
    .then((results) => {
      results.forEach((result) => {
        // do something with result
      });
    })
    .catch((error) => {
      // do something with error
    });
}
```

Reply:
nit: would be nice to add the usage cases in the comments. idk if JS does similar things to java docs but that would be helpful for the interns too.

Reply:
I think JS does have good doc stylings, will add them